src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import 'blocks/card';
+@import 'blocks/stars';
+
 body {
   margin: 0;
+  padding: 50px;
+  background-color: #f0f0f0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Font families should be stored in variables for easier maintenance and to ensure consistency across the project as recommended by the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).